.github/workflows/ci.yml

```diff
@@ -81,7 +81,7 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v3
       with:
-        go-version: "1.19"
+        go-version: "1.21"
```

Comment:
Ideally, you'd have a separate PR for the go-version bump. Similarly stuff that reformats things etc. This just makes a reviewer more focused and so a bit easier. Also if the main work is controversial the version bump is not help up.

Reply:
I know, I personally hate small PRs and decided to include it since it potentially could have adversely affected things in this PR.